### DISCUSSION TITLE
feed axis accordingly

### DISCUSSION THREAD

**Discussion by arlimbad222** [fra589/grbl-Mega-5X#237] [Ideas]
hey, all I am testing some jobs today with the rotary axis and I found some speed issues with the rotary axis because when I type (G01 X-100 A-314 F600) the f600 is for both axis linear and rotary is it any way to increase feed for that rotary axis.

here are my calculations when we say X-100 F600 the F600 is mm/min but for the A it's degree /min so I just need to increase that particular axis feed like (600*pi)=1884.

I found these issue because my design has some straight lines and some curve so when X and A
moves together machine go slow there and when only X moves machine go fast there. I attached the test job photos for reference.

correct me if I am wrong because I am not sure about my calculations 

![IMG20220116192946](https://user-images.githubusercontent.com/34487708/149674060-d062f917-e0a4-4bbc-9015-cfda0ec1e4e5.jpg)

**Comment by RaphaelDives** [comment]
Hello,
up to now I did only screws on my rotary axis. 😉
But in my opinion, the feedrate has to be diameter-dependent, right? So I would say that’s not the task of grbl, but your Cam Software… I would look there for a solution.
Cheers
Raphael

Maybe, you can just increase the Max Feedrate for your rotary axis?

**Comment by fra589** [comment]
Hi @arlimbad222, hi @RaphaelDives,  
  
When I added the extra axes to Grbl, I dealt with the subject of speeds maybe a bit too quickly...  
Maybe I should revisit this topic in light of http://linuxcnc.org/docs/2.8/html/gcode/machining-center.html#sub:feed-rate  
But right now it's hard for me to find the time for everything I have in mind :-)  
  
@++;  
Gauthier.

**Reply by arlimbad222** [reply]
Hi @fra589 

It's okay i will wait. I just have an idea to do that feed things but unfortunately I am not coding guy so there is one option to wait until the next release if you add those things please update me .

Thanks for such an nice work.
